### ISSUE TITLE
Plug-In only compatible with Skyrim Version 1.6.629

### ISSUE THREAD

**Issue by agosch** [Ryan-rsm-McKenzie/ExamplePlugin-CommonLibSSE#9]
Hi, i tried to run the sample Plug-In today with the GOG version of the game. How is it possible to upgrade this to a newer version of SKSE? The SKSE Loader is working and i can attach a debugger to it.

**Comment by Ryan-rsm-McKenzie** [comment]
It should be as simple as updating the copy of commonlib to the latest

**Comment by agosch** [comment]
I checked out a fresh copy today and recompiled everything. 
My steps for the plugin were:

1. git checkout https://github.com/Ryan-rsm-McKenzie/ExamplePlugin-CommonLibSSE
2. git submodule init
3. git submodule update
4. cmake --preset vs2022-windows -DCOPY_OUTPUT=ON
5. Opening the generated solution file and build common and the plugin (both worked)

My steps for the skse loader were:

1. Cloning commons and skse64
2. On skse64 i switchted to the branch "gog"
3. Build both with cmake

Maybe I overlooked some obvious step here.
